### PR TITLE
Add a Jetpack Sync action for theme installation

### PR DESCRIPTION
**Pull request by gititon**
Adds a Jetpack Sync action for theme installation

#### Changes proposed in this Pull Request:

Adds a Jetpack Sync action for theme installation via the addition of jetpack_installed_theme action

#### Testing instructions:

phpunit to run new test for theme install sync

#### Proposed changelog entry for your changes:

jetpack_installed_theme action added upon theme installation

### CODE REVIEW

**Comment by lezama** [comment]
Great!

File: tests/php/sync/test_class.jetpack-sync-themes.php [hunk 1]
```diff
@@ -71,6 +87,42 @@ public function test_changed_theme_is_synced() {
...
+				'slug'   => $theme_stylesheet,
+			)
+		);
+
+		if ( is_wp_error( $api ) ) {
+			wp_die( $api );
+		}
+
+		//$upgrader = new Theme_Upgrader( new Theme_Upgrader_Skin( compact('title', 'nonce', 'url', 'theme') ) );
+
```
Comment (enejb) [inline tests/php/sync/test_class.jetpack-sync-themes.php]: lets remove comment or the space.

Comment (gititon) [inline tests/php/sync/test_class.jetpack-sync-themes.php]: Done!

**Comment by enejb** [comment]
Looks great!

**Pull request merged**